### ISSUE TITLE
typeerror: Property value expected type of string but got null

### ISSUE THREAD

**Issue by harrymash2006** [facebook/react-native#37549]
### Description

anyone got any solution: I am getting error like this too but for react-native components like ActivityIndicator, Button, KeyboardAvoidingView and error keeps popping up randomly from other components.

for example this kind of error: typeerror: node_modules/react-native/Libraries/Components/ActivityIndicator/ActivityIndicator.js: Property value expected type of string but got null

And the interesting thing is this error is occurring on iOS only, Android works just fine. Anyone got any idea ?

I tried doing all the things, yarn reset cache and re-installing all the node_modules but nothing works.

### React Native Version

0.64.0

### Output of `npx react-native info`

System:
    OS: macOS 13.4
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
    Memory: 1.07 GB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 8.5.5 - ~/.nvm/versions/node/v16.15.0/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    Android SDK:
      API Levels: 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.1, 30.0.2, 30.0.3, 31.0.0, 32.0.0, 33.0.0
      System Images: android-21 | Google APIs Intel x86 Atom, android-22 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom_64, android-24 | Android TV Intel x86 Atom, android-24 | Intel x86 Atom, android-24 | Google Play Intel x86 Atom, android-25 | Google APIs ARM EABI v7a, android-25 | Google APIs Intel x86 Atom, android-25 | Google Play Intel x86 Atom, android-26 | Google Play Intel x86 Atom, android-27 | Intel x86 Atom, android-27 | Intel x86 Atom_64, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-28 | Google Play Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom, android-29 | Google APIs Intel x86 Atom_64, android-29 | Google Play Intel x86 Atom, android-29 | Google Play Intel x86 Atom_64, android-30 | Google Play Intel x86 Atom, android-31 | Google APIs Intel x86_64 Atom, android-31 | Google Play Intel x86 Atom_64, android-33 | Google APIs Intel x86 Atom_64, android-N | Android Wear ARM EABI v7a, android-N | Android Wear Intel x86 Atom, android-N | Intel x86 Atom, android-N | Intel x86 Atom_64
      Android NDK: 22.1.7171670
  IDEs:
    Android Studio: 2021.3 AI-213.7172.25.2113.9014738
    Xcode: 14.3/14E222b - /usr/bin/xcodebuild
  Languages:
    Java: 18.0.2 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1 
    react-native: 0.64.0 => 0.64.0 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

Just try to run the app on iOS and it shows this error after js bundle is downloaded 100%. This does this after app.js is mounted

### Snack, code example, screenshot, or link to a repository

<img width="1417" alt="Screenshot_2023-05-24_at_6_06_35_PM" src="https://github.com/facebook/react-native/assets/5957516/06652f34-9206-4ff0-8c75-56ffb245b2a1">

**Comment by yoel-wiply** [comment]
I'm, experiencing this issue too! Let me know if you figure it out!

**Comment by yoel-wiply** [comment]
I actually resolved this issue by upgrading my EXPO SDK to version 48. Hope this helps!

**Comment by harrymash2006** [comment]
Ok but are you using an ejected react native app or expo only ?

**Comment by cortinico** [comment]
> 0.64.0

Closing as you're on a too old version of React Native and we can't offer support or bugfixes on this version sadly.

**Comment by harrymash2006** [comment]
@cortinico which version do you support for bug fixes ? Please let me try so I can try to upgrade to that version and see if this issue still persists

**Comment by Symous** [comment]
@harrymash2006 any solution here?

**Comment by Symous** [comment]
@cortinico Hi, can you please give me some advices about how to position the error reason of this issue? this error occurs in iOS only while working fine in Android. I have no idea about this and spent lots of time to fix this, thanks a lot.

**Comment by PirunSeng** [comment]
I think I've got a familiar issue:
```
"node_modules/react-native/Libraries/Vibration/Vibration.js: /path-to-my-project/node_modules/react-native/Libraries/Vibration/Vibration.js: Property value expected type of string but got null"
```

React 19.0.0
ReactNative 0.79.1

I'm in the middle of tweaking this with a patch. Yet, there would be a stable version resolving this issue or some other better solution.

**Comment by TaurusWood** [comment]
I have a similar problem:
`node_modules/react-native/Libraries/Lists/FlatList.js: Property value expected type of string but got null` in the terminal.

However, after some testing, I found it's actually not an issue. The error happened because I opened the JS bundle directly in Chrome at http://localhost:8081/rn/index.bundle.

Then I realized that the JS bundle can only run properly in the iOS environment.
